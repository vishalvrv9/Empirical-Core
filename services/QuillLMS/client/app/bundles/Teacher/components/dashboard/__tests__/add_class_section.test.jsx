import React from 'react';
import { shallow } from 'enzyme';

import AddClassSection from '../add_class_section';

describe('AddClassSection component', () => {

  it('should render', () => {
    expect(shallow(<AddClassSection />)).toMatchSnapshot();
  });

});
