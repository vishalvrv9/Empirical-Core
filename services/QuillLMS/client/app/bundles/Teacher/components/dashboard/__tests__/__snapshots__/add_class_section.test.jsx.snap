// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddClassSection component should render 1`] = `
<a
  href="/teachers/classrooms/new"
>
  <div
    className="dashed"
  >
    <img
      className="plus-icon"
      src="/add_class.png"
    />
    <h3>
      Add a Class
    </h3>
  </div>
</a>
`;
